js/ui/form/ui.form.layout_manager.js

```diff
@@ -148,6 +148,18 @@ const LayoutManager = Widget.inherit({
         return dataField ? this.option('layoutData.' + dataField) : null;
     },
 
+    _isCheckboxUndefinedStateEnabled: function(editorOption) {
+        if(editorOption.allowIndeterminateState !== true || editorOption.editorType !== 'dxCheckBox') {
```

Comment:
imho the inverted notation is easy to read:

if(editorOption.allowIndeterminateState === true && editorOption.editorType === 'dxCheckBox') 

change?
imho the additional indent for entire function body is ok.

Reply:
Fixed